internal/common/common.go

```diff
@@ -183,10 +198,15 @@ func fromAmber(provenance *types.ValidatedProvenance) (*ProvenanceIR, error) {
 		return nil, fmt.Errorf("could get builder image digest from *amber.ValidatedProvenance: %v", err)
 	}
 
+	repoURIs := []string{}
+	repoURIs = append(repoURIs, amber.GetConfigSourceURI(predicate))
+	repoURIs = append(repoURIs, amber.GetMaterialsGitURI(predicate)...)
```

Comment:
If the RepoURI appears both in `ConfigSource` and in `Materials` then `repoURIs` will contain two copies, right? That is probably ok, but can you add a comment explaining it?

Reply:
Thanks for adding the comment. I am still a bit uncomfortable with it. Also the comment does not explain the _why_, which is the most important thing you want from a comment. 

Do you expect the source URIs to be different or formatted differently in materials and in config source? I am asking because, a provenance should have exactly one source URI. 

Note that in the amber format we don't set the config source, and only set the Repo URI in materials. This is already the case in `[testdata/provenance.json](https://github.com/project-oak/transparent-release/blob/main/testdata/provenance.json).` So `GetConfigSourceURI` is not needed.

In generic slsa ([example](https://paste.googleplex.com/5755513575833600)), the same URI is included both in materials and in config source. So in this case too `GetConfigSourceURI` is not needed, because it is redundant.